stats/view/worker.go

```diff
@@ -371,15 +371,7 @@ func (w *worker) toMetric(v *viewInternal, now time.Time) *metricdata.Metric {
 		return nil
 	}
 
-	var startTime time.Time
-	if v.metricDescriptor.Type == metricdata.TypeGaugeInt64 ||
-		v.metricDescriptor.Type == metricdata.TypeGaugeFloat64 {
-		startTime = time.Time{}
-	} else {
-		startTime = w.startTimes[v]
```

Comment:
The only place we read from this map now is https://github.com/census-instrumentation/opencensus-go/blob/5fa069b99bc903d713add0295c7e0a55d34ae573/stats/view/worker.go#L350
Maybe rename it to `viewStartTimes` to make it more clear?